src/threadlist.cpp

```diff
@@ -686,7 +687,7 @@ ThreadList::postRestart(double readTime, int restartPause)
   TLSInfo_RestoreTLSState(motherofall);
 
   restartPauseLevel = restartPause;
-  DMTCP_RESTART_PAUSE(1);
```

Comment:
Yes, we already used '1'.  Originally, there was no DMTCP_RESTART_PAUSE statement in `mtcp_restart.c`.  They were all in `theadlist.cpp`.  In commit 7b8164, you had updated some code, and you had then inserted a duplicate `DMTCP_RESTART_PAUSE(1);` statement in mtcp_restart.c, that shadowed the existing `DMTCP_RESTART_PAUSE(1);` statement in threadlist.cpp.

This PR is now renumbering the `DMTCP_RESTART_PAUSE` statements of threadlist.cpp, starting at '2', so that we can keep the statment with '1' that was added to mtcp_restart..c.